addons/test_mail/tests/test_mail_composer.py

```diff
@@ -209,6 +209,37 @@ def test_mail_composer_mass_wtpl(self):
         self.assertEqual(composer_form.composition_mode, 'mass_mail')
         self.assertEqual(composer_form.model, self.test_records._name)
 
+    @users('employee')
+    def test_mail_composer_template_switching(self):
+        """ Ensure that the current user's identity serves as the sender,
+        particularly when transitioning from a template with a designated sender to one lacking such specifications.
+        Moreover, we verify that switching to a template lacking any subject maintains the existing subject intact. """
+        # Setup: Prepare Templates
+        template_complete = self.template.copy({
+            "email_from": "not_current_user@template_complete.com",
+            "subject": "subject: template_complete",
+        })
+        template_no_sender = template_complete.copy({"email_from": ""})
+        template_no_subject = template_no_sender.copy({"subject": ""})
+        form = Form(self.env["mail.compose.message"])
+
+        for composition_mode in ["comment","mass_mail"]:
```

Comment:
I feel like you don't use a linter as you regularly have small linting issues, do you ? In 15 the ci/style is not running but you will end up with a red flag for 16.0... so check your linter (or setup one if you don't).
```suggestion
        for composition_mode in ["comment", "mass_mail"]:
```